srcpkgs/bash-language-server/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'bash-language-server'
+pkgname=bash-language-server
+version=4.8.3
+revision=1
+archs="i686 x86_64"
+hostmakedepends="jq yarn"
+depends="nodejs"
+short_desc="Bash language server"
+maintainer="sirkhancision <jsantiago12tone@gmail.com>"
+license="MIT"
+homepage="https://github.com/bash-lsp/bash-language-server"
+distfiles="https://github.com/bash-lsp/bash-language-server/archive/refs/tags/server-${version}.tar.gz"
+checksum=a6c367e19a007b889f59fa7aa6a8c4fb0fae83fbfcc928e7a8f4366015dedbd6
+
+do_build() {
+	yarn
+	yarn run compile
+}
+
+do_install() {
+	TARGET_PATH="usr/lib/${pkgname}"
+	cd server
+
+	cp package.json{,.bak}
+	read -ra devDependencies < <(jq -r '.devDependencies | keys | join(" ")' package.json)
+	yarn remove --frozen-lockfile "${devDependencies[@]}"
+	mv package.json{.bak,}
+
+	vmkdir ${TARGET_PATH}
+	vcopy * ${TARGET_PATH}
+
+	vmkdir usr/bin
+	ln -sf /${TARGET_PATH}/out/cli.js ${DESTDIR}/usr/bin/${pkgname}
+	chmod 0755 ${DESTDIR}/${TARGET_PATH}/out/cli.js
+
+	vlicense ${wrksrc}/LICENSE
```

Comment:
```suggestion
	vlicense LICENSE
```

Reply:
This is because of the previous `cd` used, meaning that the license is at the child directory.

Reply:
../LICENSE then